dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -712,13 +712,13 @@ private String getCursorValue(final ConnectionContext state, final CursorBasedRe
         return "0";
     }
 
-    private List<OperationResult<ScanResult<String>>> scatterGatherScan(final CursorBasedResult<String> cursor, final String... pattern) {
+    private List<OperationResult<ScanResult<String>>> scatterGatherScan(final CursorBasedResult<String> cursor, final int count, final String... pattern) {
```

Comment:
Yes, but only for dyno-client. The `count` parameter is part of the redis api and is supported by jedis. Without this change the default of 10 is always used.